packages/console/src/components/CreateTenantModal/SelectTenantPlanModal/SkuCardItem/index.tsx

```diff
@@ -52,7 +52,7 @@ function SkuCardItem({ sku, onSelect, buttonProps }: Props) {
           </div>
         </div>
         <div className={styles.description}>
-          <PlanDescription skuId={skuId} planId={skuId} />
+          <PlanDescription skuId={skuId} />
```

Comment:
It seems that the <SkuCardItem /> component is intended to display comparisons of quota for three different plans: dev, free, and pro. From the definition of the <PlanDescription /> component, it appears that the `isEnterprisePlan` input parameter might influence the displayed content. We may need to determine within the <PlanDescription /> component whether the tenant is an enterprise plan based on the skuId.

Reply:
Always rely on `isEnterprisePlan` as the single source of truth to determine whether the provided skuId corresponds to an enterprise plan in the frontend.
For example, legacy plans may not be included in the currently publicly available SKUs, but they should not be treated as enterprise plans either.